src/Microsoft.Identity.Web.TokenCache/IMsalTokenCacheProvider.cs

```diff
@@ -26,6 +26,13 @@ public interface IMsalTokenCacheProvider
         /// <param name="tokenCache">Token cache for which to initialize the serialization.</param>
         void Initialize(ITokenCache tokenCache);
 
+        /// <summary>
```

Comment:
Adding a method to an interface sadly breaks backward compat. 

Solutions: 

1. https://devblogs.microsoft.com/dotnet/default-implementations-in-interfaces/ (doesn't work on .net classic though)
2. extension methods
3. options object

In this case, could the safety flag be passed via the constructor of `MsalAbstractTokenCacheProvider` instead?